engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -224,6 +278,17 @@ public Vector3f closestPointOnAABBToPoint(Vector3f p) {
         return r;
     }
 
+    /**
+     * Sweeps an AABB on to the AABB in the given direction and returns the normal of the first plane of intersection.
+     *
```

Comment:
@flo and I double-checked this method, it includes some weird behavior, e.g. if all flags are set the normal will be overridden and only the value for Z will be returned.
Besides that there is no single use of the method in the engine and also none in any module.

Please remove the entire method ;)